### PR TITLE
Even more compact node design

### PR DESCRIPTION
**Pull request by joeyballentine**
- Shortened all user inputs
- Removed even more spacing, giving a more unified design
- Slider design improvement

image

Further improvements and design changes can come in a later PR, I think this is good for now

### CODE REVIEW

**Comment by RunDevelopment** [comment]
Iterators still have the old design
image

**Comment by RunDevelopment** [comment]
Sliders look a bit weird if you move the track all the way to one end.
image

File: src/renderer/components/node/NodeOutputs.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ import { PyTorchOutput } from '../outputs/PyTorchOutput';
 
 interface FullOutputProps extends Omit<Output, 'id' | 'type'>, OutputProps {
     definitionType: Type;
+    index: number;
+    length: number;
```
[the remark this answers is not in the document]
Comment (joeyballentine) [reply src/renderer/components/node/NodeOutputs.tsx:20]: Wtf i thought i got rid of all of those

**Comment by joeyballentine** [comment]
> Sliders look a bit weird if you move the track all the way to one end

@RunDevelopment that's just a chakra problem. It was even worse before i made it more rectangular. I'm not well really sure how to fix that without constraining the input, which is bound to break something. I'd rather just have the small visual weirdness for now

**Comment by RunDevelopment** [comment]
> I'd rather just have the small visual weirdness for now
That's fair.

**Comment by joeyballentine** [comment]
also, i broke light mode lol

**Comment by joeyballentine** [comment]
there we go, should be good now

**Review by RunDevelopment — APPROVED** [review]

**Pull request merged**